src/AdminService.Data/Interfaces/IServiceConfigurationRepository.cs

```diff
@@ -17,5 +17,7 @@ public interface IServiceConfigurationRepository
     Task<List<Guid>> EditAsync(List<Guid> servicesIds);
 
     Task<bool> DoesAppInstalledAsync();
+
+    Task<DbServiceConfiguration> GetServiceAsync(Guid serviceId);

```

Comment:
you are in the serviceConfigurationRepository, it is clear that you get Service data, rename it to GetAsync